tests/plugins/tab/errorinlineeditor.js

```diff
@@ -15,8 +15,6 @@
 			var editor = this.editor;
 
 			editor.focus();
-			var sel = editor.getSelection();
-			sel.removeAllRanges();
```

Comment:
This test does not seem to test anything sensible after this change – it basically just presses the <kbd>Tab</kbd> key now. Additionally, the unmodified version of the test raises the very same error described in [https://dev.ckeditor.com/ticket/17028](https://dev.ckeditor.com/ticket/17028) – which is the issue tested here. Could you check if the issue reappeared after your changes and think if you can come up with some sensible way to test it?